packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.ts

```diff
@@ -0,0 +1,27 @@
+import { observable } from "@microsoft/fast-element";
+import { DelegatesARIALink, Anchor } from "../anchor";
+import { StartEnd } from "../patterns/index";
+import { applyMixins } from "../utilities/apply-mixins";
+
+/**
+ * A Breadcrumb Item Custom HTML Element.
+ *
+ * @public
+ */
+export class BreadcrumbItem extends Anchor {
+    /**
+     * @internal
+     */
+    @observable
+    public showSeparator: boolean = true;
```

Comment:
@khamudom a bit of a nit, but in preference of brevity here, could `separator` work just as well here? Technically, this renders the separator - so it's less about conditionally showing this and more about whether it is present (still true/false).

Reply:
updated to `separator`